modules/file/5.46/overlay/BUILD.bazel

```diff
@@ -0,0 +1,87 @@
+load("@aspect_bazel_lib//lib:expand_template.bzl", "expand_template")
+
+expand_template(
+    name = "gen_magic_h",
+    out = "src/magic.h",
+    template = "src/magic.h.in",
+    substitutions = {
+        "X.YY": "546",
+    },
+)
+
+cc_library(
```

Comment:
should this not be public? (I understand that the `cc_shared_library` is public)